exercise-3/exercise-3-table/exercise-3-table.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Pomodoro Timer - zsoltnagy.eu</title>
+    <link rel="stylesheet" href="node_modules/normalize.css/normalize.css">
+    <link rel="stylesheet" href="./exercise-3-table.css">
+  </head>
+  <body>
+    <div class="container">
+      <table>
+        <thead>
+          <tr>
+            <th>Task name</th>
+            <th>Status (done / planned)</th>
+            <th>Controls</th>
+          </tr>
+        </thead>
+        <tbody class="js-task-table-body">
+        </tbody>
+      </table>
+      <form class="js-add-task"
+        action="javascript:void(0)">
```

Comment:
why need set action attributes?

Reply:
i think set the action attribute for a submit form is necessary to define where the form data should be sent and processed. In that code there is no URL so i replace it with javascript:void(0)

Reply:
It's not correct. You should use `event.preventDefault` in the submit event